src/main/daml/Daml/Finance/Interface/Settlement/Types.daml

```diff
@@ -15,3 +17,27 @@ data Step = Step
     quantity : Instrument.Q
       -- ^ The instrument and amount to be transferred.
   deriving (Eq, Show)
+
+-- | Describes an allocation of an `Instruction`.
+data Allocation
```

Comment:
I think we should use CamelCasing for enums, but ok to do this in a separate PR.

Reply:
maybe something for the style guide, seems like we are inconsistent there. will add a ticket for that as well.